translations/es/status.yml

```diff
@@ -3,3 +3,4 @@ overall_reporting_status: Estado general de informes
 reported_online: Reportado en línea
 reporting_status: Estado de informes
 statistics_in_progress: Estadísticas en progreso
+status_by_goal: Estado por objetivo
```

Comment:
A (non-native) spanish speaker said it should be "estados por objetivos" (note plural in estados). But I'd want verification.